rocAL/rocAL_pybind/examples/rocAL_api_tf_detection_pipeline.py

```diff
@@ -112,14 +112,13 @@ def main():
                                                       random_aspect_ratio=[0.8, 1.25],
                                                       random_area=[0.1, 1.0],
                                                       num_attempts=100,path = image_path)
-        resized = fn.resize(decoded_images, resize_x=300, resize_y=300)
-        pipe.set_outputs(resized)
+        resized = fn.resize(decoded_images, resize_width=300, resize_height=300)
+        pipe.setOutputs(resized)
     pipe.build()
     image_iterator = ROCALIterator(pipe, device=device)
 
     cnt = 0
     for i, (images_array, bboxes_array, labels_array, num_bboxes_array) in enumerate(image_iterator, 0):
-        images_array = cp.transpose(images_array, (0, 2, 3, 1)) if device == "gpu"  else np.transpose(images_array, [0, 2, 3, 1])
```

Comment:
Check if the cupy import is still necessary

Reply:
cupy is needed for the GPU backend